.github/workflows/test.yml

```diff
@@ -2,8 +2,11 @@
 on:
   push:
     branches:
-      - staging
+      - release-v*
       - trying
+      - staging
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+' # this only concerns tags on stable
```

Comment:
are you sure that this regex match only stable tags ?
https://regex101.com/r/d4jF4S/1

shouldn't it at least be '^v[0-9]+.[0-9]+.[0-9]+$' ?

Reply:
https://gist.github.com/jhorsman/62eeea161a13b80e39f5249281e17c39 😋

Reply:
https://regex101.com/r/KuQoOu/2/

Reply:
this is not exactly like regex:
- it does not support `^` and `$`. 
- `.` are not wildcards
- it works as match and not find

source: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet.
I had the same concerns as you, but after experimenting a bit with it, it really only matches stables release tags

Reply:
they'll do everything differently....... 😞